### DISCUSSION TITLE
Substitute Trakt uses for SIMKL

### DISCUSSION THREAD

**Discussion by umbrellaplug** [umbrellaplug/umbrellaplug.github.io#411] [Ideas]
Current list of things Trakt is being used for within Umbrella:

1. Bookmarks - tvshowtitle, title, resume_id, imdb, tmdb, tvdb, season, episode, genre, mpaa, studio, duration, percent_played, paused_at
2. Lists - Liked Lists, User Lists, Public Lists
3. Movie and TV Collection
4. Movie and TV Watchlists
5. Watched Items Movies, Shows, Seasons

**Comment by masyk** [comment]
Simkl Dev here.

Simkl has standard user Watchlists ( for TV Shows, Movies, Anime) :

API for Watchlists https://simkl.docs.apiary.io/#reference/sync  returns all ids including TMDB, TVDB, IMDB. When using `extended=full` also return list of all episodes.
Default Watchlists on Simkl: 

- Watching (For TV Shows and Anime only)
- Plan to watch
- On hold (For TV Shows and Anime only)
- Completed
- Dropped

More info: https://docs.simkl.org/how-to-use-simkl/basic-features/watchlists-and-custom-lists

Welcome to chat on Discord, DM at any time for detailed questions: https://discord.gg/u89XfYn

**Reply by umbrellaplug** [reply]
@Adamosborne83 Do you have SIMKL? Can you go ahead and sign up so when I get ready to test features you'll be able to?

Don't you have some large lists? I'd be interested to see how simkl does with importing your stuff and if it can be done on a free tier or requires vip.

**Reply by adamosborne83** [reply]
> @Adamosborne83 Do you have SIMKL? Can you go ahead and sign up so when I get ready to test features you'll be able to?
> 
> Don't you have some large lists? I'd be interested to see how simkl does with importing your stuff and if it can be done on a free tier or requires vip. 

I think I do have an account, when you/ we initially added it. I will check though.

I do indeed have some lists :-)
10 TV lists
12 Movies lists
Largest list has over 5000 movies. 
Happy to test as always 😀

**Reply by masyk** [reply]
> @adamosborne83 Do you have SIMKL? Can you go ahead and sign up so when I get ready to test features you'll be able to?
> 
> Don't you have some large lists? I'd be interested to see how simkl does with importing your stuff and if it can be done on a free tier or requires vip.

You can import watch history from many sources including trakt here: https://simkl.com/apps/import/

**Reply by hcgiub001** [reply]
SIMKL has a loyalty programme
[Use Simkl for 20 days per month and get the PRO status next month](https://simkl.com/vip/loyalty/)
Here is a list of pro features
https://simkl.com/vip/
The one that interest me the most is
30 custom lists (2k each) total 60k

So if you sync with SIMKL and use kodi 20 days out of a month, you basically get free PRO, pretty much everyone uses kodi everyday

In comparison Trakt offers
 10 personal lists (with up to 1,000 items each)  , no anime
 
 EDIT: I just realised another thing trakt changed, I'm on VIP and when I check what none VIP members get on this site
 https://trakt.tv/vip/manage
I get this
![image](https://github.com/user-attachments/assets/a69b0099-6691-405f-b74d-d2d5089c4cc0)

If I load this same page in incognito (no VIP)
I get this
![image](https://github.com/user-attachments/assets/f5d8a576-59fe-4cba-87df-a7f520e5dbbb)

seems like they've lowered the custom lists much further
you have 5 custom lists with 500 each totalling 2.5k

**Reply by adamosborne83** [reply]
> > @adamosborne83 Do you have SIMKL? Can you go ahead and sign up so when I get ready to test features you'll be able to?
> > 
> > Don't you have some large lists? I'd be interested to see how simkl does with importing your stuff and if it can be done on a free tier or requires vip.
> 
> You can import watch history from many sources including trakt here: https://simkl.com/apps/import/

Hi,

Thanks for your reply.

Couple things I have noticed initially. 

I understand Ads might be needed for free tier, but 5 full screen pop up adds within 20mins.

When I import my watchlist, it will import items from Trakt TVShows that I have removed all viewing history from. Simkl still shows it.

Possibly having list import become dynamic? So it automatically updates? 

It seems very orientated towards TVShows and Animie, as opposed to movies, is there a reason behind this?

I will keep playing, I use trakt almost daily, previously their free version and now their VIP.

**Reply by adamosborne83** [reply]
Also, no ability to import my lists from trakt. 
I understand constraints with Trakt. I will keep playing.

**Reply by masyk** [reply]
> Also, no ability to import my lists from trakt. I understand constraints with Trakt. I will keep playing.

To clarify, https://simkl.com/apps/import/trakt/ imports your Watch history for Watchlists. Individual Custom Lists can be imported inside the Custom List in the Import tab. You can follow other's lists instead of creating your own or create Auto Custom Lists https://docs.simkl.org/how-to-use-simkl/basic-features/watchlists-and-custom-lists/creating-and-managing-custom-lists/auto-lists.

There is no api for Custom Lists yet, as it works on V2 Beta which is currently in development. Watchlists has API https://simkl.docs.apiary.io/#reference/sync

**Reply by adamosborne83** [reply]
Free version does not allows lists to be created either?

**Reply by masyk** [reply]
> I understand Ads might be needed for free tier, but 5 full screen pop up adds within 20mins.

Ads are set up to show max once a day in Adsense, not sure why you saw more. 

> It seems very orientated towards TVShows and Animie, as opposed to movies, is there a reason behind this?

Simkl has all the TV Shows, all Anime and all Movies, there is no prefference towards one or the other, https://simkl.com/lists/discover/?type=all&official=true&sort=popularity,desc&franchises=true

Here's how Simkl manages it's data for all 3 https://docs.simkl.org/how-to-use-simkl/faq/frequently-asked-questions/how-to-add-new-shows-missing-episodes-and-movies-to-simkl

**Reply by masyk** [reply]
> Free version does not allows lists to be created either?

Custom list use V2 Beta engine. V2 Beta is currently available for PRO and VIP members while it's in development.

**Reply by adamosborne83** [reply]
> > I understand Ads might be needed for free tier, but 5 full screen pop up adds within 20mins.
> 
> Ads are set up to show max once a day in Adsense, not sure why you saw more. 
> 
> > It seems very orientated towards TVShows and Animie, as opposed to movies, is there a reason behind this?
> 
> Simkl has all the TV Shows, all Anime and all Movies, there is no prefference towards one or the other, https://simkl.com/lists/discover/?type=all&official=true&sort=popularity,desc&franchises=true
> 
> Here's how Simkl manages it's data for all 3 https://docs.simkl.org/how-to-use-simkl/faq/frequently-asked-questions/how-to-add-new-shows-missing-episodes-and-movies-to-simkl

Two adds within 2 mins

![Screenshot_20241001_163625_Discord.jpg](https://github.com/user-attachments/assets/119c40d2-078b-4656-bc5f-e91cf7d7b03d)

![Screenshot_20241001_163619_Discord.jpg](https://github.com/user-attachments/assets/1d65f6ca-2365-49df-81c5-d7b86e652587)

**Reply by masyk** [reply]
Might be that you're browsing in anonimously, constantly opening an embed new anonimous session in new tab? Or have some cookie blocker installed.

**Comment by adamosborne83** [comment]
Just for super clarity, if i come across negative in anyway, shape or form, this is not intentional, I have nothing but admiration for what people create and what it can do to help people with their viewing needs.

I use numerous services and always look to help and support, and where needed give feedback.

We need to remember that 95% of people will never sign up or pay. So need to see all the differences.

**Reply by masyk** [reply]
> Just for super clarity, if i come across negative in anyway, shape or form, this is not intentional, I have nothing but admiration for what people create and what it can do to help people with their viewing needs.
> 
> I use numerous services and always look to help and support, and where needed give feedback.
> 
> We need to remember that 95% of people will never sign up or pay. So need to see all the differences.

no worries, love tracking down bugs and help with setup.
